Home/home.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>OnyxShop</title>
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons+Outlined"
+      rel="stylesheet">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet">
+    <link rel="stylesheet" href="https://onyx-ui.netlify.app/css/style.css">
+    <link rel="stylesheet" href="../css/navigation.css">
+    <link rel="stylesheet" href="../css/main.css">
+</head>
+<body>
+    <header>
+        <nav class="navbar">
+            <section class="brand">
+                <span class="brand-name"><a href="#">OnyxShop</a></span>
+            </section>
+            <section class="searchbar">
+                <div class="input-icon">
+                    <i class="material-icons">search</i>
+                    <input  class="input" type="text" placeholder="Search">
+                </div>
+            </section>
+            <section class="nav-links">
+                <button class="btn">Login</button>
+                <div class="badge">
+                    <i class="material-icons-outlined">favorite_border</i>
+                    <div class="number dnd">0</div>
+                </div>
+                <div class="badge">
+                    <i class="material-icons-outlined">shopping_cart</i>
+                    <div class="number dnd">0</div>
+                </div>
+            </section>
+        </nav>
+    </header>
+    <main>  
+        <div class="men-card-container">
+            <div class="img-overlay-card">
+                <p class="overlay-text">Men</p>
+            </div>
+            <div class="img-overlay-card">
+                <p class="overlay-text">Men</p>
+            </div>
+            <div class="img-overlay-card">
+                <p class="overlay-text">Men</p>
+            </div>
+            <div class="img-overlay-card">
+                <p class="overlay-text">Men</p>
+            </div>
+            <div class="img-overlay-card">
+                <p class="overlay-text">Men</p>
+            </div>
+        </div>
+        <div class="banner">
+            <img src="../assets/banner.jpg" alt="banner">
+        </div>
+        <div class="collection-card">
+            <div class="card-container">
+                <div class="card-img-container">
+                    <img src="../assets/products/summer collection.jpg" alt="summer collection">
```

Comment:
I think according to your folder structure you can directly go like this -->  /assets/producuts/summer_collection.png
& try to avoid spacing in names

Reply:
going like this caused issues in my component library so that is why going like this now.

Reply:
cool then you can go ahead in merging looks good overall